HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -48,6 +48,7 @@ def self.calculate(runner, weather, hpxml_bldg, cfa, hvac_systems, update_hpxml:
       apply_hvac_heat_pump_logic(hvac_sizing_values, hvac_cooling)
       apply_hvac_equipment_adjustments(mj, runner, hvac_sizing_values, weather, hvac_heating, hvac_cooling, hvac_system)
       apply_hvac_installation_quality(mj, hvac_sizing_values, hvac_heating, hvac_cooling)
+      apply_autosizing_factors(hvac_sizing_values, hvac_heating, hvac_cooling)
```

Comment:
Let's use a consistent method name:
```suggestion
      apply_hvac_autosizing_factors(hvac_sizing_values, hvac_heating, hvac_cooling)
```